tests/config_module_no_underscore_munge/chain.js

```diff
@@ -0,0 +1,34 @@
+/* @flow */
+
+class A {
+  _property1: number;
+  static _sProperty: number;
+
+  constructor() {
+    this._property1 = 5;
+  }
+  _method1(): number {
+    return 1;
+  }
+  static _sMethod(): string {
+    return "some string";
+  }
+}
+A._sProperty = 48;
+
+class B extends A {
+  _property1: string;
+  static _sProperty: string;
+
+  constructor() {
+    //super();
```

Comment:
Why commented? It's actually a bug that Flow allows this (which should hopefully be fixed soon...but might as well anticipate the fix here)